web/packages/hovercards/src/style.scss

```
@@ -179,11 +222,135 @@ $color-blue: #1d4fc4;
 
 	.gravatar-hovercard__inner {
 		align-items: center;
-		justify-content: start;
+		justify-content: center;
 		gap: 34px;
 	}
 
 	.gravatar-hovercard__error-message {
 		color: $color-gray;
 	}
 }
+
+.gravatar-hovercard__drawer {
+	position: absolute;
+	top: 0;
+	left: 0;
+	bottom: 0;
+	right: 0;
+	visibility: hidden;
+	overflow: hidden;
+	z-index: 1;
+
+	.gravatar-hovercard__drawer-backdrop {
+		position: absolute;
+		top: 0;
+		left: 0;
+		bottom: 0;
+		right: 0;
+		background-color: rgba($color-black, 0.4);
+		opacity: 0;
+		transition: opacity 0.3s ease-in-out;
+	}
+
+	.gravatar-hovercard__drawer-card {
+		position: absolute;
+		left: 0;
+		bottom: 0;
+		width: 100%;
+		max-height: 100%;
+		display: flex;
+		flex-direction: column;
+		background-color: $color-white;
+		border-top-left-radius: 4px;
+		border-top-right-radius: 4px;
+		padding: 20px 0;
+		box-sizing: border-box;
+		transform: translateY(100%);
```

Utilize the GPU to ensure smooth animations:

```suggestion
		transform: translate3d(0, 100%, 0);
```